addons/event_sale/models/event.py

```diff
@@ -25,6 +29,11 @@ def _default_tickets(self):
         except ValueError:
             return self.env['event.event.ticket']
 
+    @api.one
+    @api.depends('event_ticket_ids.seats_max')
+    def _compute_seats_max(self):
+        self.seats_max = sum(ticket.seats_max for ticket in self.event_ticket_ids)
```

Comment:
@antonylesuisse , I checked again , and  you are right. using read_group we can directly get sum of seats_max. so we wil use it.
THanks.